services/src/main/scala/cromwell/services/metadata/impl/archiver/ArchiveMetadataSchedulerActor.scala

```diff
@@ -62,9 +63,14 @@ class ArchiveMetadataSchedulerActor(archiveMetadataConfig: ArchiveMetadataConfig
   private val workflowArchiveTotalTimeMetricPath: NonEmptyList[String] = archiverMetricsBasePath :+ "workflow_archive_total_time"
   private val workflowsToArchiveMetricPath: NonEmptyList[String] = archiverMetricsBasePath :+ "workflows_to_archive"
 
+  private val TerminalWorkflowStatuses: List[String] = List(WorkflowSucceeded, WorkflowAborted, WorkflowFailed).map(_.toString)
+
   // kick off archiving immediately
   self ! ArchiveNextWorkflowMessage
 
+  // schedule for workflows left to archive metric
+  context.system.scheduler.schedule(archiveMetadataConfig.backoffInterval, archiveMetadataConfig.instrumentationInterval)(workflowsLeftToArchiveMetric())
```

Comment:
It's unlikely at 1 minute, but if the query is super slow, and if we set this value very short, these might start overlapping and building up faster than they're completed.

It might be better to trigger the next query 1 minute _after_ the first query completes?

Reply:
Hmm, I see what you mean. That makes sense. I can update it to schedule the next query a minute after the previous one completes 👍